### PR TITLE
Rahti2 migration

### PR DESCRIPTION
**Pull request by trispera**
## Proposed changes

Rahti 2 production docs:

https://csc-guide-preview.rahtiapp.fi/origin/Rahti2-migration/

Closure of the previous [PR](https://github.com/CSCfi/csc-user-guide/pull/1870)

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [ ] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

**Review by tolonenj — APPROVED** [review]

File: docs/cloud/tutorials/pod-affinity.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Pod (anti) affinity
+
+Pod affinity and anti-affinity allow you to constrain where your Pods will be scheduled on, based on the labels of Pods already running on that node. This general idea can also be used to select (or avoid) specific nodes with specific hardware configurations like a GPU.
+
+This is useful because sometimes (affinity) it is interesting to be sure that two Pods share the same mounted volume and avoid network synchronization issues, or the opposite, (anti-affinity) make sure two pods will not share the same server and thus increase the availability of an application.
+
+In this example of a deployment we want the Pods from the `wes-deployment` to run in the same node as the `celelery-worker` ones so they can both mount the same storage volume (which can only be mounted on a single physical node).
```
Comment (tolonenj) [inline docs/cloud/tutorials/pod-affinity.md:7]: typo: celery

File: docs/cloud/tutorials/pod-affinity.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
++                - celery-worker
++            topologyKey: "kubernetes.io/hostname"
+       initContainers:
+```
+
+The `affinity` block needs to be under `spec`. For applying this to another deployment one will need to change only the `value` and perhaps the `key`, so it corresponds to one `key` and `value` pair of the **Labels** belonging to the Pod we want to have affinity with. It is possible to see the **Labels** of a Pod via the command line or via the web interface.
+
+![Labels](../img/celery-worker.png)
+
+When the `affinity` block in added to this deployment, the scheduler will (re-)launch the Pods corresponding to `wes-deployment` into the same as `celery-worker` is running. As the policy used is `requiredDuringSchedulingIgnoredDuringExecution`, if there is no space in the chosen node, the deployment will not succeed. Other policy is `preferredDuringSchedulingIgnoredDuringExecution`, which means that the scheduling will not fail if there is no space in the chosen node as they will be scheduled in a different one.
```
Comment (tolonenj) [inline docs/cloud/tutorials/pod-affinity.md:35]: in? Or should it be 'is'

**Review by tolonenj — APPROVED** [review]
Nice - good work!!

**Review by tolonenj — APPROVED** [review]

**Review by tolonenj — APPROVED** [review]

**Review by tolonenj — APPROVED** [review]
LGTM

**Review by tolonenj — APPROVED** [review]

**Review by tolonenj — APPROVED** [review]

File: docs/cloud/tutorials/pod-affinity.md (hunk 2 above)
Comment (lvarin) [reply docs/cloud/tutorials/pod-affinity.md]: solved, thanks

File: docs/cloud/tutorials/pod-affinity.md (hunk 1 above)
Comment (lvarin) [reply docs/cloud/tutorials/pod-affinity.md]: solved, thanks

**Review by tolonenj — APPROVED** [review]

**Review by tolonenj — APPROVED** [review]

**Review by tolonenj — APPROVED** [review]

**Review by lvarin — APPROVED** [review]
LGTM

File: csc-overrides/assets/snippets/rahti2_announcement.md
Comment (Oussalvatore) [inline csc-overrides/assets/snippets/rahti2_announcement.md:1]: The date for shutting down old Rahti will change

File: docs/cloud/rahti2/rahti-what-is.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* Deploy a web application written in common languages like Python, Javascript or Java to the web with just one command.
+* If you want to run a web application or host a web site, Rahti 2 will likely be the platform for you. It comes with a lot of the most common features needed for web based applications.
+
+## OpenShift OKD v Kubernetes
+
+OpenShift [OKD](https://www.okd.io/) is optimized for multi-tenant deployment. This means that different tenants will be sharing the same hardware. For this reason, **privileged mode** is not allowed, and containers **cannot be run as root**.
+
+OpenShift provides these extra services over a standard Kubernetes offering:
+
+* Web UI: <https://landing.2.rahti.csc.fi/>
```
Comment (Oussalvatore) [inline docs/cloud/rahti2/rahti-what-is.md:25]: Old rahti landing page will be replaced with the new rahti 2 landing page. Maybe it is better to point the url to rahti.csc.fi

File: docs/cloud/rahti/security-guide.md [hunk 4]
```diff
@@ -1,27 +1,26 @@
-# Security guide
-
-Rahti applications are exposed to the Internet, and
+--8<-- "rahtibeta_announcement.md"
+Rahti 1 applications are exposed to the Internet, and
 their security should be treated with an appropriate care.
-The user on whose account a service is running in Rahti is
+The user on whose account a service is running in Rahti 1 is
 responsible for its security. See the [terms of use](https://rahti.csc.fi/agreements/terms_of_use/) for details about the
```
Comment (Oussalvatore) [inline docs/cloud/rahti/security-guide.md:5]: Rahti 2 has new terms of use: https://wiki.csc.fi/ContainerPlatforms/RahtiServiceTermsofUse

Comment (lvarin) [reply docs/cloud/rahti/security-guide.md:5]: I guess that the link in the documentation will still be valid, as you are going to update the terms of use page? Or will you publish these new terms of use in a new public link?

Comment (Oussalvatore) [reply docs/cloud/rahti/security-guide.md:5]: The content of https://rahti.csc.fi will be replaced by the content of https://landing.2.rahti.csc.fi/. The problem now is that  https://landing.2.rahti.csc.fi/ has no section for terms of use, and instead refers to the old terms of use (under Service description). So yes we need to host the new terms of user somewhere else. Is it fine to add a page for them in Rahti documentation (probably the easiest solution)?

Comment (lvarin) [reply docs/cloud/rahti/security-guide.md:5]: IMHO, feel free to publish the terms anywhere. 

I think it is easier that you publish it in <https://landing.2.rahti.csc.fi/agreements/terms_of_use/> that will be later published in the old link. But as it is you who wrote them, you decide.

File: docs/cloud/rahti2/usage/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+network and is isolated from other projects. Users can only see projects
+they have created themselves or that have been shared with them. Any
+containers, volumes and other resources created by users are always created
+within a project.
+
+You can either select applications to run in the application catalog that is
+visible when first logging in or launch any applications using the
+primitives described in the [background](../concepts.md) chapter.
+
+Log in at <https://landing.2.rahti.csc.fi/> (see [Getting access](../access.md)
```
Comment (Oussalvatore) [inline docs/cloud/rahti2/usage/getting_started.md:14]: The landing page will be available in both https://rahti.csc.fi and https://landing.2.rahti.csc.fi/ . which one is better to have in the docs?

File: docs/cloud/rahti/usage/projects_and_quota.md [hunk 6]
```diff
@@ -37,20 +36,20 @@ For more information about using the web interface, refer to the
...
+    Rahti 1 can be used free of charge for open research and
     education in Finnish universities and polytechnics.
 
 Any usage within OpenShifts projects are billed
 to the billing unit quota of the CSC project selected. Note that the selected CSC project must
-have Rahti access and the user creating a project must be a member of that computing project,
+have Rahti 1 access and the user creating a project must be a member of that computing project,
 otherwise the OpenShift project creation will fail.
 
 * If you would like to know which CSC computing projects you are a member of, you
```
Comment (Oussalvatore) [inline docs/cloud/rahti/usage/projects_and_quota.md:55]: default billing projects are not supported in Rahti-2

File: docs/cloud/rahti/usage/projects_and_quota.md [hunk 7]
```diff
@@ -65,7 +64,7 @@ explicitly specify one.
 * If you would like to know which CSC computing project an OpenShift project is
 associated with, you can do so using the _oc_ command line tool. You can find
 instructions for setting up oc in the [command line tool usage
-instructions ](/cloud/rahti/usage/cli). For example, if your OpenShift project is called
+instructions ](cli.md). For example, if your OpenShift project is called
 *my-openshift-project*, you would run:
 
 ```bash
```
Comment (Oussalvatore) [inline docs/cloud/rahti/usage/projects_and_quota.md:70]: Project labels are visible in WUI as well !!!

File: docs/cloud/rahti/usage/projects_and_quota.md (hunk 6 above)
Comment (lvarin) [reply docs/cloud/rahti/usage/projects_and_quota.md:55]: fixed

File: docs/cloud/rahti2/usage/getting_started.md (hunk 5 above)
Comment (lvarin) [reply docs/cloud/rahti2/usage/getting_started.md]: fixed

**Pull request merged**